server/src/modules/reviews/mappers/review-map.ts

```diff
@@ -0,0 +1,68 @@
+import { EntityRepository } from '@mikro-orm/core'
+import { Result } from '../../../shared/core/result'
+import { UniqueEntityID } from '../../../shared/domain/unique-entity-id'
+import { PlaceEntity } from '../../../shared/infra/db/entities/places/place.entity'
+import { ReviewEntity } from '../../../shared/infra/db/entities/review.entity'
+import { Review } from '../domain/entities/review'
+import { ReviewDTO } from './review-dto'
+
+export class ReviewMap {
```

Comment:
I guess my only concern if the scenario where someone is onboarding to the codebase and doesn't know what Mikro is. The token `Persistence` is immediately obvious.

Reply:
That's a good point. At the same time, `Mikro` should be in the name somewhere, because the method is specifically for persistence via Mikro -- we can't persist to a text file using that method. How about to/from`MikroPersistence`?